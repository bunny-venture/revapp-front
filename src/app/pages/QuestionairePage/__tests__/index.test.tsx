import * as React from 'react';
import { render } from '@testing-library/react';

import { QuestionnairePage } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<QuestionnairePage  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<QuestionnairePage />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
