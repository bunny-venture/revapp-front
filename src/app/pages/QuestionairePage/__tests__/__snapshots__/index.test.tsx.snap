// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuestionnairePage  /> should match snapshot 1`] = `null`;
